alpha/lib/model/entry.php

```diff
@@ -335,17 +345,96 @@ public function incViews ( $should_save = true )
 	{
 		myStatisticsMgr::incEntryViews( $this );
 	}
-
+	
+	public static function getMultiLingualSupportedFields()
+	{
+		return self::$multiLingualSupportedFields;
+	}
+	
+	public function alignFieldValue($field, $value)
+	{
+		$supportedFields = self::getMultiLingualSupportedFields();
+		switch ($field)
+		{
+			case $supportedFields[self::NAME]:
+				return kString::alignUtf8String($value, self::MAX_NAME_LEN);
```

Comment:
wouldn't this chop the value of the fields in custom data in case they have more then 40 chars?

Reply:
it is the same limitation imposed earlier